### DISCUSSION TITLE
7602031K6 (Hue Go Lights) not exposing colors ?

### DISCUSSION THREAD

**Discussion by R-Nabil** [Koenkk/zigbee2mqtt#22212] [Devices]
Hi,

I used to have this device in HA through ZHA, and I used to be able to change the color of the light as well (blue, red, green etc).
I moved everything to zigbee2mqtt and this is somehow not showing colors ? I can't seem to be able to change it.

Can anyone help me figure out why ?

Thks

**Comment by R-Nabil** [comment]
Hi. Not sure if this was missed or not enough details provided ?

**Comment by Koenkk** [comment]
Fixed!

Changes will be available in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) in a few hours from now.
